### ISSUE TITLE
Show log level options in help and documentation

### ISSUE THREAD

**Issue by Schniz** [Schniz/fnm#523]
tl;dr -- we can use [`Arg::possible_values`](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.possible_values) to limit the number of inputs allowed, and it will also show them in the `--help` docs.

First, let's add a `LogLevel::possible_values()` function to the `impl` block:
https://github.com/Schniz/fnm/blob/0b13e2c8cd4415a7575f9ba765fcc7782c1481a9/src/log_level.rs#L8-L37
The function should return an array of strings. We have 4 possible values, so the return value should be `&'static [&'static str, 4]`. Maybe there's a nicer way of marking the return value less explicit, i don't know 🤷 

Now, we can go to the loglevel declaration in config.rs:

https://github.com/Schniz/fnm/blob/0b13e2c8cd4415a7575f9ba765fcc7782c1481a9/src/config.rs#L39-L47

Here we can add to the `#[structopt(...)]` macro declaration the `possible_values` attribute: `#[structopt(..., possible_values = LogLevel::possible_values()]`

After that, we can clean up #516, because Clap will generate the proper error for us anyway.
